src/openrct2/CmdlineSprite.cpp

```diff
@@ -262,7 +262,7 @@ static bool is_changable_pixel(sint32 palette_index) {
 }
 
 static sint32 get_closest_palette_index(sint16 *colour){
-    uint32 smallest_error = -1;
+    uint32 smallest_error = (uint32)-1;
```

Comment:
Sorry, but why are you storing -1 in an unsigned integer?

Reply:
Sawyer did this all the time.

Reply:
Sawyer didn't use C... I'd be more comfortable with making the 0xFFFFFFFF more explicit, somehow, rather than forcing the conversion. We have a define for it, surely?

Reply:
Not specifically. We have named ones, like RIDE_INDEX_NULL.